pages/viewPlans/index.js

```diff
@@ -11,7 +11,7 @@ export default function ViewAllPlans() {
   return (
     <div>
       <p>{`There are`} {data.length} {data.length === 1 ? 'template' : 'templates'} {`in your vault`}</p>
-      {data.map(plan => <div key={plan._id}><Link href={`viewPlans/${plan._id}`}>{plan.name}</Link></div>)}
+      {data.map(plan => <TemplateCard key={plan._id} data={plan} />)}
       <button type="button" onClick={() => router.push(`/createPlan`)}>New Template</button>
```

Comment:
I think if the main purpose of this button is to redirect the user then we probably want it to be a link rather than a button. Something like. 

``` js
<Link href="/createPlan">New Template</Link>
```

Reply:
I'm on it.